### ISSUE TITLE
Expired signed Key for Linux installation. Could you update? 

### ISSUE THREAD

**Issue by ehcatl** [MiKTeX/miktex-packaging#280]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex-packaging/blob/master/CONTRIBUTING.md)

Hi everyone: 

The signed key at the ubuntu repositories is no longer active, so apt-update cannot load the repository. 

Could you update the key?

**Comment by edocevoli** [comment]
Hm, the key is valid until April 2023. Please show the error message.

**Comment by ehcatl** [comment]
Here's the error feed. The error is at the end of the process.

```
jaime@Ehcatl:~$ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D6BC243565B2087BC3F897C9277A7293F59E4889
[sudo] password for jaime: 
Executing: /tmp/apt-key-gpghome.h21Xr4T9de/gpg.1.sh --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D6BC243565B2087BC3F897C9277A7293F59E4889
gpg: key 277A7293F59E4889: public key "MiKTeX Packager <packager@miktex.org>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: invalid key resource URL '/tmp/apt-key-gpghome.h21Xr4T9de/home:manuelschneid3r.asc.gpg'
gpg: keyblock resource '(null)': General error
gpg: key 1488EB46E192A257: 1 signature not checked due to a missing key
gpg: key 7721F63BD38B4796: 2 signatures not checked due to missing keys
gpg: key 1488EB46E192A257: 1 signature not checked due to a missing key
gpg: key 3B4FE6ACC0B21F32: 3 signatures not checked due to missing keys
gpg: key D94AA3F0EFE21092: 3 signatures not checked due to missing keys
gpg: key 871920D1991BC93C: 1 signature not checked due to a missing key
gpg: Total number processed: 8
gpg:       skipped new keys: 8
jaime@Ehcatl:~$ echo "deb http://miktex.org/download/ubuntu focal universe" | sudo tee /etc/apt/sources.list.d/miktex.list
deb http://miktex.org/download/ubuntu focal universe
jaime@Ehcatl:~$ sudo apt-get update
Hit:1 http://mx.archive.ubuntu.com/ubuntu focal InRelease
Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]                                                           
Get:3 http://mx.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]                                                                                                                                               
Hit:4 https://dl.google.com/linux/chrome/deb stable InRelease                                                                                                                                                                       
Get:6 http://mx.archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]                                                                                                                                                        
Hit:7 http://download.opensuse.org/repositories/home:/manuelschneid3r/xUbuntu_21.10  InRelease             
Hit:8 http://ppa.launchpad.net/wereturtle/ppa/ubuntu focal InRelease                                       
Get:9 http://security.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [40.8 kB]
Get:10 http://security.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [66.3 kB]
Get:11 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 DEP-11 Metadata [2 464 B]
Get:12 http://mx.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [1 789 kB]
Get:5 https://ctan.javinator9889.com/systems/win32/miktex/setup/deb focal InRelease [2 030 B]
Get:13 http://mx.archive.ubuntu.com/ubuntu focal-updates/main i386 Packages [650 kB]
Get:14 http://mx.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [278 kB]
Get:15 http://mx.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [391 kB]
Get:16 http://mx.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 DEP-11 Metadata [944 B]
Get:17 http://mx.archive.ubuntu.com/ubuntu focal-backports/main amd64 DEP-11 Metadata [9 576 B]
Get:18 http://mx.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [30.8 kB]
Err:5 https://ctan.javinator9889.com/systems/win32/miktex/setup/deb focal InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 277A7293F59E4889
Reading package lists... Done       
W: GPG error: https://ctan.javinator9889.com/systems/win32/miktex/setup/deb focal InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 277A7293F59E4889
E: The repository 'http://miktex.org/download/ubuntu focal InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
jaime@Ehcatl:~$ 
```

**Comment by edocevoli** [comment]
Thank you. So the public `277A7293F59E4889` key is successfully imported and then `apt-get` cannot find/use it. Sorry, but I have no idea why this happens and I am not able to fix it.
